.travis.yml

```diff
@@ -23,7 +23,7 @@ script:
   - (cd doc/; make latexpdf html)
   - ./ci/build-qemu-powernv.sh
   - wget https://openpower.xyz/job/openpower/job/openpower-op-build/label=slave,target=witherspoon/lastSuccessfulBuild/artifact/images/witherspoon.pnor
-  - PATH=$(pwd)/qemu:$PATH ./op-test --bmc-type qemu --qemu-binary `pwd`/qemu/ppc64-softmmu/qemu-system-ppc64 --host-pnor witherspoon.pnor --run-suite qemu
+  - PATH=$(pwd)/qemu/build:$PATH ./op-test --bmc-type qemu --qemu-binary `pwd`/qemu/build/qemu-system-ppc64 --host-pnor witherspoon.pnor --run-suite qemu
```

Comment:
Why? Something changed in recent version. If so you should call out details in patch description. Otherwise it becomes hard to understand.

-Vasant

Reply:
yes, In recent upstream version the binary files are store in to different folder structure.  Hence the change. As you mentioned I will add the details to the up coming patches.

As this patch is for one for CI itself. I will merge the same.

Regards
Hariharan T.S.